src/nvim/eval.c

```diff
@@ -20263,10 +20265,12 @@ static inline Job *common_job_start(JobOptions opts, typval_T *rettv)
       xfree(opts.term_name);
       free_term_job_data(data);
     } else {
-      EMSG(_(e_jobexe));
+      EMSG3(_(e_jobexe), progname, uv_strerror(rettv->vval.v_number));
```

Comment:
The third argument here is always `-1` and is not the uv error. The value returned in case of a failed spawn could be used as error status from `job_start()`.